Error while `go get`

`createJenkins` has changed in upstream Jenkins library bndr/gojenkins#91 and this causes errors while installing:
```
→ go get -u github.com/dihedron/terraform-provider-jenkins
# github.com/dihedron/terraform-provider-jenkins
../../go/src/github.com/dihedron/terraform-provider-jenkins/config.go:19:35: cannot use c.ServerURL (type string) as type *http.Client in argument to gojenkins.CreateJenkins
```

--- a/config.go
+++ b/config.go
@@ -16,7 +16,7 @@ type Config struct {
 
 func (c *Config) getAPIClient() (*jenkins.Jenkins, error) {
 
-	client := jenkins.CreateJenkins(c.ServerURL, c.Username, c.Password)
+	client := jenkins.CreateJenkins(nil, c.ServerURL, c.Username, c.Password)
 	if c.CACert != "" {
 		// provide CA certificate if server is using self-signed certificate
 		client.Requester.CACert, _ = ioutil.ReadFile(c.CACert)

